atlp-rwanda/vikings-ec-bn
# #184347046 prompting users to change there password

#### What does this PR do?
Forcing manual password update
#### Description of Task to be completed?
The task was about sending a notification to the user when the password spends some time without updating it.
#### How should this be manually tested?
1. Clone [repo](https://github.com/atlp-rwanda/vikings-ec-bn.git)
2. Change into the directory vikings-ec-bn
3. Checkout to this branch: `feature/prompt-user-to-change-their-password-184347046`
4. In your `.env` file, create a `PASSWORD_EXPIRATION_IN=2m , CRON_PERIOD=1m`
5. Run the command `npm install` to install all packages
6. Run `npm run reset:db` to reset database
7. Rum `npm run dev`
8. clone front-end testing project [socket-testing](https://github.com/ndatumuremyi/socket-test-andela.git)
9. run `npm install`
10. run `npm run start`
11. you can add a user and verify him.
12. after 2m as specified in .env you will see the notification from user 
note : server must be running on 5000 port 
#### What are the relevant pivotal tracker stories?
[184347046](https://www.pivotaltracker.com/story/show/184347046)
